_posts/2023-10-18-distro-puppet-acceptance-tests.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: Acceptance testing using distro Puppet versions
+date: 2023-10-18
+github_username: ekohl
+category: changelog
+---
+
+Vox Pupuli's CI now supports testing using distro Puppet versions.
+At the moment this is used for ArchLinux, Debian 12 and Fedora 37 & 38.
+
+Within Vox Pupuli we maintain a lot of modules.
+Keeping those healthy is a challenge and having working acceptance tests makes that feasible.
+For each operating system we need two things: a working (container) image and Puppet.
+The former is generally easy (except for operating systems with licenses), but for the we need some system packages.
+Until recently we've relied Puppet's [RPM](https://yum.puppet.com/) and [deb](https://apt.puppet.com/) packages.
```

Comment:
`we've relied Puppet's` or `we've relied on Puppet's`?

Reply:
Also, line 15, "but for the we"